src/Illuminate/Auth/GenericUser.php

```diff
@@ -131,4 +131,14 @@ public function __unset($key)
     {
         unset($this->attributes[$key]);
     }
+
+    /**
+     * Get the column name for the "password".
+     *
+     * @return string
+     */
+    public function getPasswordName()
```

Comment:
Why are you duplicating this here?

Reply:
This is here because he added this method to the interface which this generic user implements.